articles/communication-services/concepts/call-automation/call-automation.md

```diff
@@ -62,7 +61,7 @@ The following list presents the set of features that are currently available in
 
 ## Architecture
 
-Call Automation uses a REST API interface to receive requests and provide responses to all actions performed within the service. Due to the asynchronous nature of calling, most actions will have corresponding events that are triggered when the action completes successfully or fails.
+Call Automation uses a REST API interface to receive requests and provide responses for all actions performed within the service. Due to the asynchronous nature of calling, most actions will trigger corresponding events upon successful completion or failure.
```

Comment:
acrolinx will complain about using future tense - try to use present tense where possible.